### PR TITLE
Added machine specific PLA material profile for BIQU B1

### PR DESCRIPTION
**Pull request by looxonline**
This is the tweaked materials file for the PR below. It allows a specific temp to be set for the B1 machine when using PLA.

https://github.com/Ultimaker/Cura/pull/8276

### CODE REVIEW

**Comment by nallath** [comment]
You also need to increase the version of the file by at least 1

**Comment by looxonline** [comment]
Will bump the version now and resubmit. Thanks for the feedback.

**Review by nallath — APPROVED** [review]

**Comment by konskarm** [comment]
I notice that https://github.com/Ultimaker/Cura/pull/8276 contains two machine definitions (biqu_b1 & biqu_b1_abl), while you change the material settings in the generic_pla_175 only for biqu_b1 in this PR. Is this intentional? Shouldn't you also add the biqu_b1_abl there?

File: generic_pla_175.xml.fdm_material [hunk 1]
```diff
@@ -27,6 +27,13 @@ Generic PLA 1.75mm profile. The data in this file may not be correct for your sp
         <setting key="surface energy">100</setting>
         <setting key="build volume temperature">28</setting>
 
+        <machine>
+            <machine_identifier manufacturer="BIQU" product="biqu_b1" />
```
Comment (konskarm) [inline generic_pla_175.xml.fdm_material:31]:
```suggestion
            <machine_identifier manufacturer="BIQU" product="biqu_b1" />
            <machine_identifier manufacturer="BIQU" product="biqu_b1_abl" />
```

**Comment by looxonline** [comment]
> I notice that [Ultimaker/Cura#8276](https://github.com/Ultimaker/Cura/pull/8276) contains two machine definitions (biqu_b1 & biqu_b1_abl), while you change the material settings in the generic_pla_175 only for biqu_b1 in this PR. Is this intentional? Shouldn't you also add the biqu_b1_abl there?

100%. Thanks for picking that up. I only added the ABL version two nights ago after finding that so many people were struggling after having added the optional ABL add on to their printers.

**Review by konskarm — APPROVED** [review]

**Pull request merged**